app/components/govuk_component/summary_list_component/card_component.rb

```diff
@@ -17,6 +18,10 @@ def default_attributes
     { class: "#{brand}-summary-card" }
   end
 
+  def heading_tag
```

Comment:
I think it might make sense to do a check here to make sure the provided level is a number from `1..6`. Something similar happens in [the accordion component](https://github.com/x-govuk/govuk-components/blob/main/app/components/govuk_component/accordion_component.rb#L35).

Reply:
@peteryates Good shout. I've updated to include this